src/AppBundle/Controller/AdminScoreController.php

```diff
@@ -105,8 +105,10 @@ public function resultAction($project, $username)
                 AND log_namespace=0
                 AND log_deleted=0 and log_user_text=:username
         UNION
-        SELECT 'blocks' as source, COUNT(*) as value FROM $loggingTable
-            WHERE log_type='block' AND log_action='block' AND log_namespace=2 AND log_deleted=0 AND log_title=:username
+        SELECT 'blocks' as source, COUNT(*) as value FROM $loggingTable l
+            INNER JOIN $userTable u ON l.log_user = u.user_id
+            WHERE l.log_type='block' AND l.log_action='block'
+            AND l.log_namespace=2 AND l.log_deleted=0 AND u.user_name=:username
```

Comment:
I am not sure what you mean. Is there any chance you can make the necessary changes?